### PR TITLE
docs(guidelines): re-introducing Postman

### PR DESCRIPTION
**Pull request by aigelonic**
Re-introducing Postman guidelines.
This time taking into account that Postman has become "cloud-only" and giving guidance on what this could entail.

There are several considerations to do when handling sensitive data and how to set up the workspaces.
Adding a section regarding how one _could_ set up Postman in a CI/CD pipeline.

### CODE REVIEW

File: docs/guidelines/postman.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+|Credentials stored unencrypted|Storing credentials in plain-text fields, in local plain-text files|Use proper solutions when handling credentials, like Postman Vault or use OAuth 2.0 authorization flows without client secrets (require user to log in through SSO)|
+
+
+## Best practices
+The automatic synchronization to the cloud imposes both security and regulationary concerns.
+Following the best practices will mitigate some of those concerns.
+
+### SSO
+_Never_ use a private or non-Equinor account in an Equinor context.
+If needing the full feature set of Postman, request access through AccessIT and use SSO for login.
```
Comment (larskaare) [inline docs/guidelines/postman.md:34]: Should we add a link to the "Enterprise log in" - https://identity.getpostman.com/enterprise/login to make things very explicit?

File: docs/guidelines/postman.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  - that access can properly be revoked
+  - that policies are followed and are enforced
+  - and others ...
+
+### Avoid storing sensitive data (in an insecure way)
+If needing to store credentials or other sensitive data, use [Postman Vault 🔗](https://learning.postman.com/docs/sending-requests/postman-vault/postman-vault-secrets/). Note that the latter is _not_ being synchronized with Postman and anything in the Vault remains local.
+
+If sensitive data is being stored in the environment, you should set the type to "secret". This will mask the input.
+
+!!! important "Be careful"
```
Comment (larskaare) [inline docs/guidelines/postman.md:47]: This is very important. Should we add a line on this in the threat model overview? Should we suggest that "people" now and then export their stuff and examine and verify what's exposed is ok?

File: docs/guidelines/postman.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+
+## Lightweight API Client
+The "old" lightweight API client, based on "Scratch Pad", has been discountinued and there is only a single official API Client that can be downloaded.
+After downloading and executing the file, there is a choice of `Or continue with the lightweight API client.` (as of April 2024, this represents itself as a small text below the login-button on starting the application). Running the application in this mode will ensure that everything stays local - nothing is being synchronized to the cloud. However, it will not be possible to use the full set of features Postman provides and the usage in this mode will be limited to simple manual API testing.
+
+!!! info
+    Using the Lightweight client is free and one does not need to apply for access in AccessIT. In many cases this will be sufficient.
```
Comment (larskaare) [inline docs/guidelines/postman.md:112]: Perhaps we should make this a bit more specific stating that it's not necessary to apply for a Enterprise account

**Review by larskaare — APPROVED** [review]
I suggest adding a TLDR at the start, 3-4 most important points

File: docs/guidelines/postman.md (hunk 1 above)
Comment (aigelonic) [reply docs/guidelines/postman.md:34]: Yes - that will be helpful for those who use the browser to login to Postman. For those using the Desktop Client the login flow will be through that.

File: docs/guidelines/postman.md (hunk 2 above)
Comment (aigelonic) [reply docs/guidelines/postman.md:47]: Yes!
Added some reflections

File: docs/guidelines/postman.md (hunk 3 above)
Comment (aigelonic) [reply docs/guidelines/postman.md:112]: 👍

**Pull request merged**